circle.yml

```diff
@@ -1,11 +1,12 @@
+machine:
+  node:
+    version: 4.2.2
+  environment:
+    awscli: $HOME/aws/bin/aws
```

Comment:
@stulees I think we don't need this as it is not deployed to amazon. right? cheers